src/main/resources/spring/migration-bean.xml

```diff
@@ -156,6 +159,10 @@
         <constructor-arg type="java.lang.String" value="index" />
     </bean>
 
+    <bean id="namespaceFile" class="java.io.File">
+        <constructor-arg type="java.lang.String" value="src/test/resources/namespaces.properties"/>
```

Comment:
Could migration-bean.xml and namespace.properties be in the same path (either "test" or "main" or something else) as they seem to serve the same purpose as example configurations.